components/tools/OmeroWeb/omeroweb/webgateway/marshal.py

```diff
@@ -134,9 +134,18 @@ def imageMarshal(image, key=None):
 
     # big images
     tiles = image._re.requiresPixelsPyramid()
-    width, height = image._re.getTileSize()
-    levels = image._re.getResolutionLevels()
-    zoomLevelScaling = image.getZoomLevelScaling()
+    rv['tiles'] = tiles
+    if (tiles):
```

Comment:
I don't know what the ``` AttributeError ``` is designed to catch, but the only code that was moved out of the try catch block are additions to the rv dict by rv.update() which cannot raise AttributeErrors.